File: rust_internal_api_system/src/grpc_key_string_maps/internal_api_service.rs
pub struct CreateAUserRequest {
}
                    
impl CreateAUserRequest {
    pub const email: &str = "email";
}
                    


pub struct CreateAUserResponse {
}
                    
impl CreateAUserResponse {
    pub const result: &str = "result";
    pub const error: &str = "error";
}
                    


pub struct DeleteAUserRequest {
}
                    
impl DeleteAUserRequest {
    pub const email: &str = "email";
}
                    


pub struct DeleteAUserResponse {
}
                    
impl DeleteAUserResponse {
    pub const result: &str = "result";
    pub const error: &str = "error";
}
                    


pub struct GetUserRequest {
}
                    
impl GetUserRequest {
    pub const email: &str = "email";
}
                    


pub struct GetUserResponse {
}
                    
impl GetUserResponse {
    pub const error: &str = "error";
    pub const user_exists: &str = "user_exists";
    pub const email: &str = "email";
    pub const username: &str = "username";
    pub const head_image: &str = "head_image";
    pub const sex: &str = "sex";
    pub const age: &str = "age";
}

File: java_free_map_backend_system/app/src/main/kotlin/grpc_key_string_maps/internal_api_service.kt

package grpc_key_string_maps

class internal_api_service_key_string_maps {
    companion object {
            class CreateAUserRequest {
                companion object {
                    @JvmField 
                    var email: String = "email"
                    val __property_list__: List<String> = listOf("email")
                }
            }

            class CreateAUserResponse {
                companion object {
                    @JvmField 
                    var result: String = "result"
                    var error: String = "error"
                    val __property_list__: List<String> = listOf("result", "error")
                }
            }

            class DeleteAUserRequest {
                companion object {
                    @JvmField 
                    var email: String = "email"
                    val __property_list__: List<String> = listOf("email")
                }
            }

            class DeleteAUserResponse {
                companion object {
                    @JvmField 
                    var result: String = "result"
                    var error: String = "error"
                    val __property_list__: List<String> = listOf("result", "error")
                }
            }

            class GetUserRequest {
                companion object {
                    @JvmField 
                    var email: String = "email"
                    val __property_list__: List<String> = listOf("email")
                }
            }

            class GetUserResponse {
                companion object {
                    @JvmField 
                    var error: String = "error"
                    var user_exists: String = "user_exists"
                    var email: String = "email"
                    var username: String = "username"
                    var head_image: String = "head_image"
                    var sex: String = "sex"
                    var age: String = "age"
                    val __property_list__: List<String> = listOf("error", "user_exists", "email", "username", "head_image", "sex", "age")
                }
            }

            class CheckIfUserExistsRequest {
                companion object {
                    @JvmField 
                    var email: String = "email"
                    val __property_list__: List<String> = listOf("email")
                }
            }

            class CheckIfUserExistsResponse {
                companion object {
                    @JvmField 
                    var error: String = "error"
                    var user_exists: String = "user_exists"
                    val __property_list__: List<String> = listOf("error", "user_exists")
                }
            }
    }
}